### PR TITLE
Lmb 812 hide iv for districts

### PR DESCRIPTION
**Pull request by JonasVanHoof**
## Description

Districts should never see the legislatuur module., they prepare it in the normal interface.

## How to test

1. does not see the menu item (main and compact menu)
2. on mandataris search page the period is not disabled
3. on organen route the `beheer fracties` are enabled
4. the mandatarissen of 2025 - heden can be edited

When in another bestuurseenheid than district the legislatuur should work as before

### CODE REVIEW

File: app/controllers/mandatarissen/persoon/mandataris.js
Comment (Rahien) [inline app/controllers/mandatarissen/persoon/mandataris.js:107]: why not add this into the if then?

File: app/routes/mandatarissen/persoon/mandataris.js
Comment (Rahien) [inline app/routes/mandatarissen/persoon/mandataris.js:42]: we can just make this a property of the session then maybe?

Comment (Rahien) [reply app/routes/mandatarissen/persoon/mandataris.js:42]: same for other cases of this pattern

File: app/services/current-session.js
Comment (Rahien) [inline app/services/current-session.js:91]: if we fetch this during load, this can be just a getter

**Review by Rahien — CHANGES_REQUESTED** [review]

**Comment by Rahien** [comment]
I'm adopting this pr while Jonas is away :)

File: app/services/current-session.js
Comment (JonasVanHoof) [inline app/services/current-session.js:48]: This is indeed a better approach 💪🏻

**Review by JonasVanHoof — COMMENTED** [review]
Looks good visibility wise But i cannot seem to  open up the beheer fracties button? It works on master, DEV and test but not in this PR :/

To go to the fracties we add the hash bestuursperiode id and this one looks ok so not sure what was changed here, any idea?

**Comment by JonasVanHoof** [comment]
> Looks good visibility wise But i cannot seem to open up the beheer fracties button? It works on master, DEV and test but not in this PR :/
> 
> To go to the fracties we add the hash bestuursperiode id and this one looks ok so not sure what was changed here, any idea?

This is because we do not have any fracties set for the period, maybe?

**Comment by bfidlers** [comment]
> > Looks good visibility wise But i cannot seem to open up the beheer fracties button? It works on master, DEV and test but not in this PR :/
> > To go to the fracties we add the hash bestuursperiode id and this one looks ok so not sure what was changed here, any idea?
> 
> This is because we do not have any fracties set for the period, maybe?

No that should not matter, I think it is because in the fractie page we don't allow selecting the bestuursperiod of a bestuursperiod with a installatievergadering that is not behandeld. I think we should remove the installatievergadering for bestuurseenheden that do not need one, with a migration.

**Review by Rahien — APPROVED** [review]
approved after review by jonas (who couldn't approve himself as he started the pr originally)

**Pull request merged**
